src/v/config/rjson_serialization.h

```diff
@@ -15,14 +15,29 @@
 #include "config/data_directory_path.h"
 #include "config/endpoint_tls_config.h"
 #include "config/seed_server.h"
-#include "config/tests/custom_aggregate.h"
```

Comment:
i didn't intend to include this commit in this PR but it still needs to go in: "core" code should not be including anything from a test directory.